core/state/statedb.go

```diff
@@ -907,7 +907,11 @@ func (s *StateDB) Commit(deleteEmptyObjects bool) (common.Hash, error) {
 	s.IntermediateRoot(deleteEmptyObjects)
 
 	// Commit objects to the trie, measuring the elapsed time
-	var storageCommitted int
+	var (
+		accounts int
+		storages int
```

Comment:
These names feel a bit strange. These reference not the actual number of accounts / storage lots committed, rather the number of trie node, right?

Reply:
yes, it refers to the nodes in account trie and storage tries. Let me fix it.